lib/api/utils/url.js

```diff
@@ -0,0 +1,48 @@
+export class UrlUtils {
+
+  /**
+   * Private
```

Comment:
please don't put private,public comments like these, they are useless. we have a convention if it starts with _ its private